migrator/src/main/java/uk/nhs/digital/ps/migrator/model/nesstar/NesstarResource.java

```diff
@@ -36,6 +24,12 @@ public boolean isLink() {
         return getUri().matches("https?://.*");
     }
 
+    public boolean isNotOnIgnoreList() {
+        // Certain links with set phrases are not required to be migrated over
+        return !StringUtils.containsIgnoreCase(getTitle(),"earlier data may be available")
+            && !StringUtils.containsIgnoreCase(getTitle(),"contact us");
```

Comment:
Can you put spaces between parameters please 🙂 
`return !StringUtils.containsIgnoreCase(getTitle(), "earlier data may be available")
             && !StringUtils.containsIgnoreCase(getTitle(), "contact us");`

Reply:
ah yes, will do